etc/nvliblist.conf

```diff
@@ -59,3 +59,12 @@ libOpenGL.so
 libvdpau_nvidia.so
 nvidia_drv.so
 tls_test_.so
+
+# put non-library file paths here (must be absolute path)
+/usr/share/glvnd/egl_vendor.d/10_nvidia.json
+/usr/share/vulkan/icd.d/nvidia_icd.json
+/usr/share/vulkan/icd.d/nvidia_layers.json
+/usr/share/vulkan/implicit_layer.d/nvidia_layers.json
+/usr/share/egl/egl_external_platform.d/15_nvidia_gbm.json
+/usr/share/egl/egl_external_platform.d/10_nvidia_wayland.json
+/usr/share/nvidia/nvoptix.bin
```

Comment:
It looks like you lost the newline at the end of the file, please add that back.

Reply:
Thanks! added